1. Provided by Wen-Yen Chuang <caleb@calno.com>
2. Update for all debian-spefific changes

--- xf86-input-tslib-0.0.6.orig/man/tslib.man
+++ xf86-input-tslib-0.0.6/man/tslib.man
@@ -22,7 +22,7 @@
 Hal is now supported and allows the touchscreen to operate
 correctly without any need to change \*q/etc/X11/xorg.conf\*q.
 .SH BASIC CONFIGURATIONS
-If hal is not available, the following defaults may be sufficient:
+If hal is not available, the following defaults may be sufficient. If you want to use hal, please comment out the Option \*qDevice\*q.
 .PP
 .nf
 .B "Section \*qInputDevice\*q"
